Modules/TOF/tofcompressed.json

```diff
@@ -0,0 +1,75 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "42",
+        "type": "2"
+      },
+      "monitoring": {
+        "url": "infologger:///debug?qc"
+      },
+      "consul": {
+        "url": "http://consul-test.cern.ch:8500"
+      },
+      "conditionDB": {
+        "url": "ccdb-test.cern.ch:8080"
+      }
+    },
+    "tasks": {
+      "TOFTaskCompressed": {
+        "active": "true",
+        "className": "o2::quality_control_modules::tof::TOFTaskCompressed",
+        "moduleName": "QcTOF",
+        "detectorName": "TOF",
+        "cycleDurationSeconds": "10",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "The other type of dataSource is \"direct\", see basic-no-sampling.json.",
+        "dataSource": {
+          "type": "direct",
+          "query_comment" : "query is in the format of binding1:origin1/description1/subSpec1[;binding2:...]",
+          "query" : "dataframe:TOF/CRAWDATA"
+        },
+        "taskParameters": {
+          "nothing": "myOwnValue"
+        },
+        "location": "remote"
+      }
+    },
+    "checks": {
+      "checkFromTOF-Diagnostic": {
+          "active": "true",
+          "className": "o2::quality_control_modules::tof::TOFCheckDiagnostic",
+          "moduleName": "QcTOF",
+          "policy": "OnAny",
```

Comment:
Meanwhile we added a detectorName to the checks. You could add it here.

Reply:
Ok, done!